### ISSUE TITLE
Unable to enable - App directory already exists

### ISSUE THREAD

**Issue by binarykitchen** [nextcloud/bookmarks#160]
### Steps to reproduce

Go to index.php/settings/users and attempt to enable Bookmarks 8.0 0.4
### Expected behaviour

It gets enabled
### Actual behaviour

It stays disabled and the warning appears `App directory already exists`
### Server configuration

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.2 LTS
Release:        14.04
Codename:       trusty
```

```
$ uname -a      
Linux binarykitchen.com 3.12.27-xenU #1 SMP Thu Aug 28 23:13:07 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

Web server:

```
$ nginx -v
nginx version: nginx/1.8.0
```

Database:

```
$ mysql --version
mysql  Ver 14.14 Distrib 5.5.43, for debian-linux-gnu (x86_64) using readline 6.3
```

PHP version:

```
$ php -v
PHP 5.5.25-1+deb.sury.org~trusty+2 (cli) (built: May 24 2015 15:45:37) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2015 Zend Technologies
    with Zend OPcache v7.0.4-dev, Copyright (c) 1999-2015, by Zend Technologies
```

ownCloud version:
ownCloud 8.0.4 (stable) 

Updated from an older ownCloud or fresh install:
Fresh install via apt-get

```
$ sudo -u www-data php ./occ app:list
Enabled:
 - activity
 - calendar
 - documents
 - files
 - files_encryption
 - files_locking
 - files_pdfviewer
 - files_sharing
 - files_texteditor
 - files_trashbin
 - files_versions
 - files_videoviewer
 - mozilla_sync
 - music
 - provisioning_api
 - tasks
 - templateeditor
 - videos
Disabled:
 - external
 - files_external
 - firstrunwizard
 - gallery
 - galleryplus
 - user_external
 - user_ldap
 - user_webdavauth
```

The content of config/config.php: (Without sensitive data)

```
$ cat config.php
<?php
$CONFIG = array (
  'instanceid' => 'oc02ca513338',
  'passwordsalt' => 'xxx',
  'datadirectory' => '/var/www/owncloud/data',
  'dbtype' => 'mysql',
  'version' => '8.0.4.2',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'owncloud',
  'dbpassword' => 'xxx',
  'installed' => true,
  'forcessl' => true,
  'loglevel' => 1,
  'logfile' => '/var/log/owncloud.log',
  'theme' => '',
  'maintenance' => false,
  'trusted_domains' => 
  array (
    0 => 'xxx.yyy.com',
  ),
  'mail_smtpmode' => 'smtp',
  'secret' => 'xxx',
  'forceSSLforSubdomains' => true,
  'mail_smtpsecure' => 'tls',
  'mail_smtphost' => '127.0.0.1',
  'mail_smtpport' => '587',
  'mail_smtpauth' => 1,
  'mail_smtpname' => 'xxx',
  'mail_smtppassword' => 'xxx',
  'mail_smtpauthtype' => 'PLAIN',
);
```

Are you using external storage, if yes which one:
None
### Client configuration

Browser:
Firefox 38

Operating system:
Ubuntu 15.04
#### Web server error log

nginx error log is empty since a couple of days, hence irrelevant.
#### ownCloud log (data/owncloud.log)

N/A
#### Browser log

Irrelevant

**Comment by blizzz** [comment]
Likely related to https://github.com/owncloud/bookmarks/issues/147

**Comment by mobarre** [comment]
Just updated to 8.1.3. Issue still exists and prevents me from reenabling my contacts app. Anything I can do ?
I got all my professional contacts in there and little time this week to debug this... Help would be appreciated ;)

**Comment by blizzz** [comment]
@mobarre this should not prevent you from dealing with contacts at all. Unless it is an issue in core.

**Comment by mobarre** [comment]
Well, for starters I'm pretty sure it prevents any carddav access unless the app is active. Anything I can do to help ?

**Comment by mobarre** [comment]
ok, forget that... it seems that enabling from the disabled app section + updating the app from there did the trick (not sure the update was necessary to make it worked).

I had no error message when activating it from there. It's at least a bit confusing :)

**Comment by marcelklehr** [comment]
Seems to have been fixed with #147
